Maquetacion_v2/index.html

```diff
@@ -10,7 +10,17 @@
 </head>
 <body>
     <section class="tools-section">
-      <h1 class="title">Front-End Tools</h1> 
+      <h1 class="title">Front-End Tools</h1>
+      <div class="tech-cards">
+        <div class="tech-card">
+          <img src="res/images/content/react_icon.svg" class="tech-card--logo">
```

Comment:
Add the "alt" attribute for the img tag and use only one "-" for the class name.